### ISSUE TITLE
Find Results with auto-refresh checkbox

### ISSUE THREAD

**Issue by JeFawk** [Scirra/Construct-feature-requests#235]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

To fix the usability as explained here: https://github.com/Scirra/Construct-bugs/issues/7982

### Possible workarounds or alternatives

Not much of a workaround or alternative apart from not using the Remove Results, and/or not adding any code with a Find Results panel active that the user is following.

### Proposed solution

Add an auto-refresh checkbox. Initially it can be set as checked by default if needed be. Unticking it would refresh the Find Results only when the user clicks on the Refresh button already present on the top side of the panel.

### Why is this idea important?

Explanation: https://github.com/Scirra/Construct-bugs/issues/7982
TLDR: Remove Result is not usable. The Panel auto-refreshes by default for no apparent reason whenever some new code is added. Very hard to use in it's current implementation related to the auto-refresh.

### Additional remarks

Shouldn't be too complex to add, and the impact it would make would be enormous.
